fabulaws-config.yml

```diff
@@ -232,6 +271,12 @@
       db-slave: 282 # 250 GB data + 32 GB swap
       web: 10 # 10 GB data (no swap)
       worker: 66 # 50 GB data + 16 GB swap
+    prod01:
+      cache: 25 # 10 GB data + 15 GB swap
```

Comment:
Just a note that these could be adjusted along with the instance sizes, as the need for swap is reduced. Deciding not to worry about it would be a perfectly reasonable decision IMO, too (unless of course the instance sizes need to increase further at any point, in which case the server creation will almost certainly break if the EBS volumes aren't increased in size).